src/structures/Interaction.js

```diff
@@ -67,6 +68,12 @@ class Interaction extends Base {
      * @type {number}
      */
     this.version = data.version;
+
+    /**
+     * This member permissions, if exists, in the channel this interaction was sent in
```

Comment:
Current description can be improved on gramatically
```suggestion
     * The permissions of the member, if one exists, in the channel this interaction was executed in```